The PostHog Glossary (#4425)
PostHog/posthog.com: contents/manual/glossary.mdx

#### Annotation
Annotations are short text notes you can apply to your data to help you understand context at a later point, or to signal important context to others. A common use for annotations it to add them shared charts and mark important events, such as feature releases or new marketing activiity. [Find out more about annotations in PostHog](/manual/annotations). 

#### App 
Apps are additional features or functionality which are built on top of PostHog. They are commonly used to integrate PostHog with other platforms, to modify events or to enhance PostHog's functionality. Apps may be built by the PostHog team, or by the community. You can [browse the app library](/apps) to see what's available, or [find out more in the docs](/docs/apps). 

#### App chain
In PostHog, the output of one app will impact the actions of the next app to create a chain. This makes it crucial to understand the order in which apps run. [Find out more about app chains in the docs](/docs/apps/build). 

#### ARR
Annual recurring revenue - basically how much revenue you expect to make each year.

> 

Reviewer: ```suggestion

#### Autocapture
The ability to capture user behavior (events, pageviews) automatically. In PostHog, uses our [JavaScript library or snippet](/docs/integrate/client/js#autocapture).
```

## B 

#### B2B
Business to business. It's when you make money by selling things or services to other businesses, rather than to individual users. Tools such as [Group Analytics](/manual/group-analytics) are especially important for B2B users in PostHog.

#### B2C
Business to customer. It's when you make money by selling things or services to individual customers, rather than to other businesses. Tools such as [experiments](/manual/experimentation) are especially important for B2C users in PostHog.

#### BAA
Business Associate Agreement, or also known as a Business Associate Contract. It's a legal document signed by contractors who may have to handle [personal health information](/manual/glossary#PHI) in order to agree to and satisfy the requirements of [HIPAA](/manual/glossary#HIPAA). [Find out more about PostHog and HIPAA](/docs/privacy/hipaa-compliance). 

## C

#### Churn
When a user stops using your product and is no longer considered to be an [active user](/manual/glossary#active-user). The speed at which users churn is known as your _churn rate_ and, if it is greater than the rate at which you acquire new customers then your product is in decline. 

#### ClickHouse

> ClickHouse is an [open source](/manual/glossary#open-source), column orientated system for managing databases. [We like it a lot and use it to power PostHog](/docs/how-posthog-works/clickhouse). 

Reviewer 2: ```suggestion
ClickHouse is an [open source](/manual/glossary#open-source), column-oriented database. [We like it a lot and use it to power PostHog](/docs/how-posthog-works/clickhouse). 
```

#### Event
Events are the core building block of PostHog. Any activity a user does, such as clicking a button or viewing a page, is an event. Events are [ingested](/manual/glossary#ingestion) into PostHog, and sometimes organized as [actions](/manual/glossary#actions).

## F 

#### Feature flag
Feature flags are used to divide users into two groups - those for whom the flag is enabled, and those for whom the flag is disabled. Based on this flag status, users are served a version of the product where a specific feature is, or isn't, available. [Find out more about feature flags in the docs](/manual/feature-flags). 

#### First-party cookies
Every time you visit a website or view content from a website, you are served a `cookie` - a data packet which contains information on your visit. First-party cookies are those which are served by the website operator and are typically used to store only user preferences. Users who self-host PostHog are able to collect additional information through their first-party cookie because PostHog is also deployed onto their servers. See also: [third-party cookies](/manual/glossary#third-party-cookies)

> 

Reviewer: ```suggestion

#### Formula
An equation or calculation that modifies data from one or more series. 

```

#### GTM
Go-to Market. In short, it's the plan you have for how you will bring a new product or feature to market, often building in intelligence on your users gained from product analytics. 

## H

#### HIPAA
Health Insurance Portability and Accountability Act. It's a piece of legislation in the US which covers, among other things, how [personal health information](/manual/glossary#PHI) must be handled. [Find out more about how to use PostHog in a which complies with HIPAA regulations](/docs/privacy/hipaa-compliance). 

## I 

#### Ingestion
The process of getting the [events](/manual/glossary#events) users trigger into PostHog, regardless of whether the data is [current](/docs/integrate/ingest-live-data) or [historic](/docs/integrate/ingest-historic-data).

#### Insights
Insights are a core feature of PostHog which enable you to query analytics and derive visualized results. [Funnels](/manual/funnels), [trends](/manual/trends), [path analysis](/manual/paths) and more are all examples of PostHog insights. 

> 

Reviewer: ```suggestion

#### Internal users
Users within your team. Can set up PostHog so their usage is [filtered out](/tutorials/filter-internal-users).

```

#### Issues
What we use to log bugs, discuss ideas and make decisions on Github. Issues are better than a [Slack](/slack) message, but worse than a [pull request](/manual/glossary#pull-request). 

## J 

## K

#### K Factor
A metric for understanding the virality of your product. You can calculate your K factor by figuring out the number of invitations or recommendations about your product which users make, then multiplying it by the success rate of those invitations or recommendations. Any result greater than 1 indicates viral growth, as every user you acquire will, on average, successfully invite another. 

#### KPI
Key Performance Indicator. A measurable metric which enables you to judge the success (or failure) of a person, team or product. Typically, teams may have multiple KPIs they are accountable for. 

## L 

#### LTV
Lifetime Value. The total amount of revenue you get from the average user before they [churn](/manual/glossary#churn), calculated using the formula `number of users / total revenue`.

## M 

#### MAU 
Monthly [Active Users](/manual/glossary#active-users). Essentially, how many users you have using your product each month. 

#### MOM

> Month-on-month. Normally this is used to track your performance from one month to the next, e.g. "We grew revenue 10% month-on-month". 

Reviewer: ```suggestion
Month-on-month or month-over-month. Normally this is used to track your performance from one month to the next, e.g. "We grew revenue 10% month-on-month". 
```

#### Open source
A model of distribution which involves giving others the ability to inspect, modify, and share what you create. PostHog is an example of both an open-source product, because [we distribute our software under an open source MIT license](https://github.com/PostHog/posthog), and [an open source company, because we work with transparency](/handbook/company/culture).

## P

#### Path
In PostHog, a path is a visualization of a users' activity achieved using the [path analysis insight](/manual/paths). Paths enable you to see, for example, all the pages which users who visited a particular page viewed afterwards. 

#### Persons
In PostHog, persons are used to aggregate the behaviour of an individual user, whether or not that user has been identified. [Find out more about Persons on PostHog](/manual/persons). 

#### PII
Personal Identifying Information. Under some regulations, such as [HIPAA](/manual/glossary#HIPAA), any information which can be used to identify a user - such as their birthday, ethnicity, or address - must be processed in accordance with law.

> 

Reviewer: ```suggestion

#### Pipeline
A managed stream of data connecting data creation, modification, storage, and access.

```

#### PHI
Personal Health Information. Regulations such as [HIPAA](/manual/glossary#hipaa) require that any information about an individual's health must be processed and store in accordance with law. 

#### Pivot
A rapid change in company direction. [One of our favourite hobbies](/blog/story-about-pivots). 

#### PQL
Product Qualified Lead. A PQL is a user who has expressed interest in your product based on actions which can be attributed to the product, such as completing a free trial or having experienced the direct value of the product.

#### Property
In PostHog, [events](/manual/glossary#events) and [persons](/manual/glossary#persons) contain additional information, called Properties. A property can be any information stored on the object, such as which device they are using or their geographic location. [Find out more about using properties in PostHog](/docs/how-posthog-works/queries). 

#### Pull request
An event when someone makes a change to a piece of software. The best thing that anyone on PostHog's team can ever be doing.  

## Q 

## R 

> 

Reviewer: ```suggestion

#### Release Conditions
The users a feature flag is active for. For example, the internal team or 50% of all users.

```

#### Retention 
In PostHog, a retention insight is used to visualize how many users return to your product after first visiting it on a particular day. [Find out more about measuring retention on PostHog](/manual/retention).

#### Returning user
In PostHog's [lifecycle insight](/manual/lifecycle) a returning user is one who did the specified action within the current period _and_ within the previous period.

#### Resurrecting user
In PostHog's [lifecycle insight](/manual/lifecycle) a returning user is one who did the specified action within the current period _but not_ within the previous period.

## S 

#### SAML
Security Assertion Markup Language is a system which enables users to login to multiple systems with one set of credentials. [Find out more about SAML on PostHog](/manual/SSO). 

> 

Reviewer: ```suggestion

#### SDK
Software development kit. Tools for building and using a software platform. PostHog has [many](/docs/integrate).

```

#### Segment
Segmentation is a way of dividing users into groups based on either demographic, geographic, behavioral or technographic properties. [Find out how to segment users with PostHog](/tutorials/how-to-segment-users). 

> 

Reviewer: ```suggestion

#### Self-host
The ability to host and manage software yourself (on your own servers). PostHog provides the ability to [self-host](/docs/self-host).

```

#### Session 
A period of time when users are [engaged](/manual/glossary#engagement) and interacting with your product. PostHog enables you to [capture sessions directly, as recordings](/manual/recordings).

> 

Reviewer: ```suggestion

#### SQL
Structured Query Language (pronounced "sequel"). A programming language for designing, building, and querying relational database management systems.

```

#### SSO 
Single Sign-on is a system which enables users to login to a system (such as PostHog) using their access to another system (such as Google Workspace) as their credentials. [Find out more about SAML on PostHog](/manual/SSO).

#### Stickiness 
In PostHog, a stickiness insight enables you to see how many times users perform an action within a set period of time. [Find out more about measuring stickiness with PostHog](/manual/stickiness). 

## T

#### Third-party cookies
Every time you visit a website or view content from a website, you are served a `cookie` - a data packet which contains information on your visit. Third-party cookies are those which are _not_ served by the website operator and are instead served by third-parties, typically advertising companies. Users who self-host PostHog are able to collect additional information through [first-party cookies](/manual/glossary#first-party-cookies) because PostHog is also deployed onto their servers.

> 

Reviewer: ```suggestion

#### Toolbar
A widget in your browser that contains easy access to useful actions or information. [PostHog's toolbar](/manual/toolbar) lets you easily inspect element, view page heatmaps, create actions, and toggle feature flags.
```

## U

#### UGC
User Generated Content refers to any content which users create within, or using your product for the purpose of sharing with others. Posts on Instagram are a typical example of user-generated content.

#### UTM 
Urchin Tracking Module. Essentially, a UTM are modifiers appended to an existing URL in order to aid attribution in marketing campaigns. [Find out more about tracking UTMs in PostHog](/tutorials/taxonomy-acquisition). 

## V

> 

Author: > 

```suggestion
#### Variants
Different versions of an experiment. In PostHog, variants are linked to [feature flags](/manual/glossary#feature-flag), which activate them.

#### VM
Virtual machine. An emulated and often isolated computing process. We use VMs to run [apps](/manual/glossary#app).
```
